impl/asm-based-transformer/src/main/java/org/wildfly/transformer/asm/ReflectionModel.java

```diff
@@ -0,0 +1,50 @@
+package org.wildfly.transformer.asm;
```

Comment:
Missing license header.

Reply:
Thanks, added and noticed why, fixed my editor settings to do this by default for Batavia project. :)